lib/fb_graph2/attribute_assigner.rb

```diff
@@ -67,6 +67,8 @@ def assign(attributes)
               end
             when :location
               Struct::Location.new raw
+            when :place
+              Place.new raw, {id: raw}
```

Comment:
This change should break GET response handling where place is a page object, not page ID.